recipes/llvmlite/meta.yaml

```diff
@@ -0,0 +1,47 @@
+{% set name = "llvmlite" %}
+{% set version = "0.20.0" %}
+{% set sha256 = "b2f174848df16bb9195a07fec102110a06d018da736bd9b3570a54d44c797c29" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+
+build:
+  number: 0
+  script: python setup.py install  --single-version-externally-managed --record=record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - enum34  # [py2k]
+    - llvmdev
+  run:
+    - python
+    - enum34  # [py2k]
+
+test:
+  imports:
+    - llvmlite
+    - llvmlite.binding
+    - llvmlite.ir
+    - llvmlite.llvmpy
+    - llvmlite.tests
+
+about:
+  home: http://llvmlite.pydata.org
+  license: BSD 2-clause
```

Comment:
Minor: this should be `BSD-2-Clause` for consistency I think; you should also add `license_family: BSD`.

Reply:
Just as FYI, we have generally been flexible as long the version of the license is specified somehow.